### PR TITLE
Add get/set_device_offset methods

### LINKED ISSUE #229: cairo_surface_set_device_offset() missing
Would be good to have a binding for [this](https://cairographics.org/manual/cairo-cairo-surface-t.html#cairo-surface-set-device-offset).

### PR DESCRIPTION
**Pull request by osa1**
Fixes #229

### CODE REVIEW

File: src/image_surface.rs [hunk 1]
```diff
@@ -100,6 +100,17 @@ impl ImageSurface {
     pub fn get_width(&self) -> i32 {
         unsafe { ffi::cairo_image_surface_get_width(self.to_raw_none()) }
     }
+
+    pub fn set_device_offset(&mut self, x_offset: f64, y_offset: f64) {
+        unsafe { ffi::cairo_surface_set_device_offset(self.to_raw_none(), x_offset, y_offset) }
+    }
+
+    pub fn get_device_offset(&self) -> (f64, f64) {
```
Comment (EPashkin) [inline src/image_surface.rs]: ImageSurface has Deref<Surface> so IMHO repeating don't needed

**Comment by EPashkin** [comment]
@osa1 Thanks.

**Comment by GuillaumeGomez** [comment]
:+1:

**Comment by sdroege** [comment]
:+1:

**Comment by sdroege** [comment]
Would be nice to also add `device_scale` and `fallback_resolution` at the same time. Very closely related and otherwise someone else will be sad that it's missing in a few weeks :)

**Comment by GuillaumeGomez** [comment]
Thanks!

@sdroege: I'll do it.

**Comment by GuillaumeGomez** [comment]
I was thinking about going to Mars but that too! :stuck_out_tongue:

**Comment by sdroege** [comment]
What do you want on Mars? There's nothing there ;)

**Comment by GuillaumeGomez** [comment]
Because I'm not there yet! :smiling_imp:

**Pull request merged**